mkdocs/livereload/__init__.py

```diff
@@ -176,7 +201,7 @@ def _serve_request(self, environ, start_response):
         # https://github.com/bottlepy/bottle/blob/f9b1849db4/bottle.py#L984
         path = environ["PATH_INFO"].encode("latin-1").decode("utf-8", "ignore")
 
-        m = re.fullmatch(r"/livereload/([0-9]+)/[0-9]+", path)
+        m = re.fullmatch(rf"{self.mount_path}livereload/([0-9]+)/[0-9]+", path)
```

Comment:
Ah I missed another aspect. This is a regex after all, so this would at least have to be `re.escape(self.mount_path)`

But it may be better to refactor it further because below we already have code that removes the mount_path prefix